src/Lumi/Components/NativeSelect.purs

```diff
@@ -108,6 +108,7 @@ styles = jss
 
           , extend: lumiInputStyles
           , padding: "8px 20px 8px 7px"
+          , paddingRight: "calc(10px + 14px) !important"
```

Comment:
Can't we instead just change the `padding` line above to `"8px 24px 8px 7px"`? How does this look on mobile btw?

Reply:
Looks fine on mobile:
<img width="381" alt="Screen Shot 2021-05-06 at 11 34 14 AM" src="https://user-images.githubusercontent.com/632981/117325867-21600880-ae5f-11eb-9132-af49f745d837.png">

And yeah I can just change the padding above (and below in the desktop mediaquery) rather than using the calc + important.